content/en/security/cspm/findings.md

```diff
@@ -61,10 +61,62 @@ The side panel lists detection rules that were evaluated against the resource, s
 
 {{< img src="security/cspm/findings/passed-rules.png" alt="Group and aggregate by resource in search" style="width:100%;">}}
 
+## Mute findings
+
+{{< callout url="" btn_hidden="true" >}}
+  Muted findings is a beta feature available to all CSPM customers. If you have any feedback or questions, contact <a href="/help">Datadog support</a>.
+{{< /callout >}} 
+
+There may be times when a finding does not match the use case for your business, or you choose to accept it as a known risk. To ignore these findings, you can mute the finding for the impacted resource. This allows you to focus on high-severity and critical findings that require your attention.
+
+For example, the ['Block Public Access' feature is enabled for S3 bucket][4] rule evaluates whether a S3 bucket is publicly accessible. If you have a S3 bucket with static assets that are meant to be publicly shared, you can mute the finding for the S3 bucket.
+
+You can mute pass/fail findings at any time. Muted findings do not impact your posture score.
```

Comment:
Does this mean the findings are still included when calculating the posture score? Or that they're omitted when calculating the score? I am guessing they're still included, in which case I would rephrase this as:
```suggestion
You can mute pass/fail findings at any time. Muting a finding does not remove it from the calculation of your posture score.
```

Reply:
Actually, it's the former (muted findings are omitted from the calculation). I'll reword this sentence to make that distinction clearer to the reader.